_data/reference.yml

```diff
@@ -25,7 +25,10 @@ toc:
     section:
     - docs/admin/accessing-the-api.md
     - docs/admin/authentication.md
-    - docs/admin/authorization.md
```

Comment:
Yes, I definitely agree you should breakout each authz plugin into its own page. My suggestions would be
1. Rename the title of the section to just "Authorization Plugins" so that "Using Authorization Plugins" isn't nested redundantly.
2. Add a "What's next" section to the bottom of index.md to point the readers to the additional documentation for each plugin.